lib/safira/accounts/attendee.ex

```diff
@@ -49,7 +50,7 @@ defmodule Safira.Accounts.Attendee do
 
   def changeset(attendee, attrs) do
     attendee
-    |> cast(attrs, [:name, :nickname, :user_id])
+    |> cast(attrs, [:name, :nickname, :user_id, :course])
```

Comment:
It would make sense to use `validate_inclusion/2` in order to ensure the course is within our courses' list.
(Same for others changesets).

Reply:
Sure but that would involve adding "None" to the list of allowed values, since we want to allow people who have no academic degree to attend the event, something that's already implemented on the frontend

Reply:
And we should also move the courses list out of the 'fake' directory if we use the values for something other that seeding the database

Reply:
From what I've tested, `validate_inclusion` does not raise nor puts an error on the changeset, if course is absent. Have you tested it?

Yes, create a folder called `data`, for example.